### PR TITLE
added configuration for OS X systems

### PR DESCRIPTION
**Pull request by ChiragGhanshani**
Connects to https://github.com/project-renard/curie/issues/95.

### CODE REVIEW

**Comment by ChiragGhanshani** [comment]
This assumes that the person already has homebrew installed. I'll just have to make this more elegant later on, but, for now, this should get yall setup. @zmughal @StayerX

**Comment by zmughal** [comment]
I'll give this a spin if the Vagrant box boots.

File: osxConfig.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+brew update
+brew install gtk\+3 gtk-doc clutter-gtk gtk-engines gtk-gnutella gtk-mac-integration gtk-murrine-engine gtk-vnc gtkdatabox gtkextra gtkglext gtkmm gtkmm3 gtksourceview gtksourceview3 gtksourceviewmm gtksourceviewmm3 gtkspell3 lablgtk pygtk pygtkglext pygtksourceview webkitgtk
+
+export PKG_CONFIG_PATH==/usr/local/Cellar/libffi/3.0.13/lib/pkgconfig/
```
Comment (zmughal) [inline osxConfig.sh:4]: Should be:

``` shell
export PKG_CONFIG_PATH=/usr/local/Cellar/libffi/3.0.13/lib/pkgconfig
```

Single equal (`=`), not a double equals (`==`).

**Comment by zmughal** [comment]
This has now been added in https://github.com/project-renard/curie/pull/96. Pulling the changes in to the Vagrant provisioning is a future step. I'll create an issue.

**Pull request closed without merging**